x-pack/legacy/plugins/apm/public/components/shared/TransactionActionMenu/sections.ts

```diff
@@ -0,0 +1,268 @@
+        query: { time }
+      }),
+      condition: !!podId
+    },
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.showPodMetricsLinkLabel',
+        { defaultMessage: 'Pod metrics' }
+      ),
+      href: getInfraHref({
+        basePath,
+        path: `/link-to/pod-detail/${podId}`,
+        query: infraMetricsQuery
+      }),
+      condition: !!podId
+    }
+  ];
+
+  const containerActions: Action[] = [
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.showContainerLogsLinkLabel',
+        { defaultMessage: 'Container logs' }
+      ),
+      href: getInfraHref({
+        basePath,
+        path: `/link-to/container-logs/${containerId}`,
+        query: { time }
+      }),
+      condition: !!containerId
+    },
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.showContainerMetricsLinkLabel',
+        { defaultMessage: 'Container metrics' }
+      ),
+      href: getInfraHref({
+        basePath,
+        path: `/link-to/container-detail/${containerId}`,
+        query: infraMetricsQuery
+      }),
+      condition: !!containerId
+    }
+  ];
+
+  const hostActions: Action[] = [
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.showHostLogsLinkLabel',
+        { defaultMessage: 'Host logs' }
+      ),
+      href: getInfraHref({
+        basePath,
+        path: `/link-to/host-logs/${hostName}`,
+        query: { time }
+      }),
+      condition: !!hostName
+    },
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.showHostMetricsLinkLabel',
+        { defaultMessage: 'Host metrics' }
+      ),
+      href: getInfraHref({
+        basePath,
+        path: `/link-to/host-detail/${hostName}`,
+        query: infraMetricsQuery
+      }),
+      condition: !!hostName
+    }
+  ];
+
+  const traceActions: Action[] = [
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.showTraceLogsLinkLabel',
+        { defaultMessage: 'Trace logs' }
+      ),
+      href: getInfraHref({
+        basePath,
+        path: `/link-to/logs`,
+        query: {
+          time,
+          filter: `trace.id:"${transaction.trace.id}" OR ${transaction.trace.id}`
+        }
+      }),
+      condition: true
+    }
+  ];
+
+  const kibanaActions: Action[] = [
+    {
+      label: i18n.translate(
+        'xpack.apm.transactionActionMenu.viewSampleDocumentLinkLabel',
+        {
+          defaultMessage: 'View sample document'
+        }
+      ),
+      href: getDiscoveryHref({
+        basePath,
+        query: getDiscoverQuery(transaction),
+        location
+      }),
+      condition: true
+    },
+    {
+      label: i18n.translate('xpack.apm.transactionActionMenu.viewInUptime', {
+        defaultMessage: 'View monitor status'
+      }),
+      href: uptimeLink,
+      condition: !!transaction.url?.domain
+    }
+  ];
+
+  const sections: Section = {
+    observability: [
```

Comment:
I guess the same concern applies here - there's an uptime link in `kibana`, even though we have an `observability` group.

Reply:
https://github.com/elastic/kibana/pull/56623#discussion_r375149494